.github/workflows/java.yml

```
@@ -102,3 +102,121 @@ jobs:
           HERE
           # Our linters are compile-time, no point re-running tests
           mvn -P errorprone install -DskipTests
+
+  java-jni-artifacts:
+    name: "JNI Libraries/${{ matrix.platform.os }} ${{ matrix.platform.arch }}"
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - { os: Linux, arch: amd64, vcpkg_arch: x64, runner: ubuntu-latest }
+          - { os: macOS, arch: arm64v8, vcpkg_arch: arm64, runner: macos-latest }
+    env:
+      VCPKG_ROOT: "${{ github.workspace }}/vcpkg"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          persist-credentials: false
+      - uses: actions/setup-java@v4
+        with:
+          cache: "maven"
+          distribution: "temurin"
+          java-version: 11
+      - name: Retrieve Go, VCPKG version from .env
+        id: vcpkg_version
+        run: |
+          vcpkg_version=$(cat ".env" | grep "VCPKG" | cut -d "=" -f2 | tr -d '"')
```

We can simplify this.

```suggestion
          vcpkg_version=$(. .env && echo "${VCPKG}")
```